packages/yew/src/scheduler.rs

```diff
@@ -25,10 +25,13 @@ struct Scheduler {
     create: Vec<Box<dyn Runnable>>,
     update: Vec<Box<dyn Runnable>>,
     render_first: VecDeque<Box<dyn Runnable>>,
+
+    #[cfg(any(feature = "ssr", feature = "render"))]
     render: RenderScheduler,
```

Comment:
Scheduler code may look a little bit messy with the feature flags, but should be mitigated with an upcoming pull request once `*Scheduler`s are replaced by BTreeMap.